src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -7,6 +7,7 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using System.Threading.Tasks;
```

Comment:
Is this using directive necessary?

Reply:
Ugh, VS keeps adding these for some reason. I missed deleting this one